code-samples/dags/airflow-kafka/produce_consume_treats.py

```diff
@@ -0,0 +1,109 @@
+"""
+### DAG which produces to and consumes from a Kafka cluster
+
+This DAG will produce messages of several elements to a Kafka cluster and consume
+them.
+"""
+
+from airflow.decorators import dag, task
+from pendulum import datetime
+from airflow.providers.apache.kafka.operators.produce import ProduceToTopicOperator
+from airflow.providers.apache.kafka.operators.consume import ConsumeFromTopicOperator
+import json
+import random
+
+YOUR_NAME = "<your name>"
+YOUR_PET_NAME = "<your (imaginary) pet name>"
+NUMBER_OF_TREATS = 5
+KAFKA_TOPIC = "my_topic"
+
+
+def prod_function(num_treats, pet_name):
+    """Produces `num_treats` messages containing the pet's name, a randomly picked
+    pet mood post treat and whether or not it was the last treat in a series."""
+
+    for i in range(num_treats):
+        final_treat = False
+        pet_moode_post_treat = random.choices(
+            ["content", "happy", "zoomy", "bouncy"], weights=[2, 2, 1, 1], k=1
+        )[0]
+        if i + 1 == num_treats:
+            final_treat = True
+        yield (
+            json.dumps(i),
+            json.dumps(
+                {
+                    "pet_name": pet_name,
+                    "pet_moode_post_treat": pet_moode_post_treat,
```

Comment:
```suggestion
                    "pet_mood_post_treat": pet_mood_post_treat,
```
Super nit picky :) but you'll need to update everywhere else too

Reply:
It is funny I had a moment of "that word looks weird but spellcheck seems to think it is fine" 🤦 (I had turned off spellcheck for screenshots two days ago)